plugins/database/postgresql/postgresql_test.go

```diff
@@ -657,3 +682,81 @@ func createTestPGUser(t *testing.T, connURL string, username, password, query st
 		t.Fatal(err)
 	}
 }
+
+func TestContainsMultilineStatement(t *testing.T) {
+	type testCase struct {
+		Input    string
+		Expected bool
+	}
+
+	testCases := map[string]*testCase{
+		"issue 6098 repro": {
+			Input:    `DO $$ BEGIN IF NOT EXISTS (SELECT FROM pg_catalog.pg_roles WHERE rolname='my_role') THEN CREATE ROLE my_role; END IF; END $$`,
+			Expected: true,
+		},
+		"multiline with template fields": {
+			Input:    `DO $$ BEGIN IF NOT EXISTS (SELECT FROM pg_catalog.pg_roles WHERE rolname=\"{{name}}\") THEN CREATE ROLE {{name}}; END IF; END $$`,
```

Comment:
Should we still be escaping double-quotes even with backticks? Same down below for `"docs example"` and `"templated field"`.

Reply:
I was just trying to be cruel to the code in the tests to make sure it still worked as expected.

Reply:
Stripped them out in https://github.com/hashicorp/vault/pull/8512/commits/405465fed2cb85612d78bc7033d99c581f4e117b